assets/css/white_boxes.css

```diff
@@ -0,0 +1,24 @@
+.sk_bg {
+    animation-duration: 1s;
+    animation-fill-mode: forwards;
+    animation-iteration-count: infinite;
+    animation-name: sk_bg_animation;
+    animation-timing-function: linear;
+    background: #f6f7f8;
+    background: linear-gradient(to right, #eee 8%, #ddd 18%, #eee 33%);
+    background-size: 800px 104px;
```

Comment:
can't use `px` - can use `rem` to make it relative.
but can't use a predefined background-size - the size of this component, especially the width, could be anything and is determined by the parent.